bitnami/apache/templates/prometheusrules.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if and .Values.metrics.enabled .Values.metrics.prometheusRule.enabled  }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  {{- if .Values.metrics.prometheusRule.namespace }}
+  namespace: {{ .Values.metrics.prometheusRule.namespace }}
+  {{- else }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: apache
```

Comment:
You're using `app.kubernetes.io/component: apache` here while using `app.kubernetes.io/component: metrics` in the **metrics-svc.yaml**. Is there a reason for that?

Reply:
yes, i choose metrics , is not realy part of apache component but it's metrics as wordpress.